julia_src/http.jl

```diff
@@ -64,50 +64,76 @@ function reopt(req::HTTP.Request)
 			)
```

Comment:
make same changes to julia_src/cbc/http.jl

Reply:
Added ability to return 400 from julia_src/cbc/http.jl in case of errors. Will also add a test case.